app/views/govuk_publishing_components/components/_details.html.erb

```diff
@@ -7,7 +7,7 @@
   css_classes << (shared_helper.get_margin_bottom)
 
   data_attributes ||= {}
-  data_attributes[:module] = 'govuk-details'
+  data_attributes[:module] = 'govuk-details gem-details'
```

Comment:
Nit: Should this be plural "modules", to indicate you can specify more than one?

Reply:
That's a good point, I'm not 100% confident of the full implications of renaming `data-module` to `data-modules`. Would it be ok with you if I record this as an issue to investigate further?

Reply:
Sure thing! Not a blocker for this PR.